docs/docs/installation/installing-superset-using-docker-compose.mdx

```diff
@@ -84,9 +84,11 @@ and wait for the `superset_node` container to finish building the assets.
 
 #### Configuring Docker Compose
 
-The following is for users who want to configure how Superset starts up in Docker Compose; otherwise, you can skip to the next section.
+The following is for users who want to configure how Superset running in Docker Compose; otherwise, you can skip to the next section.
```

Comment:
nit, should this be "runs"?
```suggestion
The following is for users who want to configure how Superset runs in Docker Compose; otherwise, you can skip to the next section.
```